src/page-objects/administration/CustomFieldDetail.ts

```diff
@@ -0,0 +1,77 @@
+import type { Page, Locator } from '@playwright/test';
+import { CustomFieldCreate } from './CustomFieldCreate';
+
+export class CustomFieldDetail extends CustomFieldCreate {
+    public readonly newCustomFieldButton: Locator;
+    public readonly customFieldEditDialog: Locator;
+    public readonly newCustomFieldDialog: Locator;
+    public readonly customFieldTechnicalNameInput: Locator;
+    public readonly customFieldPositionInput: Locator;
+    public readonly customFieldTypeSelectionList: Locator;
+    public readonly customFieldModifyByStoreApiCheckbox: Locator;
+    public readonly customFieldCancelButton: Locator;
+    public readonly customFieldAddButton: Locator;
+    public readonly customFieldEditApplyButton: Locator;
+    public readonly customFieldLabelEnglishGBInput: Locator;
+    public readonly customFieldPlaceholderEnglishGBInput: Locator;
+    public readonly customFieldHelpTextEnglishGBInput: Locator;
+    public readonly customFieldDeleteListButton: Locator;
+    public readonly customFieldDeleteDialog: Locator;
+    public readonly customFieldDeleteCancelButton: Locator;
+    public readonly customFieldDeleteButton: Locator;
+
+    constructor(public readonly page: Page) {
+        super(page);
```

Comment:
just curious, why do you use **super(page);** in some pages

Reply:
I extend from the  `CustomFieldCreate` page-objects and with `super()` I tell the root objects the extended properties.